scaling_performance/optimizing_ephemeral_storage.adoc

```diff
@@ -11,65 +11,77 @@
+* Local storage is a best effort resource.
+* Pods can get evicted due to other pods filling the local storage,
+after which, new pods are not admitted until sufficient storage has
 been reclaimed.
 
 Ephemeral storage is still exposed to pods in the same way, but there
 are new methods for implementing requests and limits on pods'
 consumption of ephemeral storage.
 
-Note that management of container logs apply only if you are using
-CRI-O as the container runtime and use file-based logging.
+[NOTE]
+====
+Management of container logs only applies if you are using
+CRI-O as the container runtime and use file-based logging for logging.
+====
 
 It is important to understand that ephemeral storage is shared among
 all pods in the system, and that {product-title} does not provide any
 mechanism for guaranteeing any level of service beyond the requests
-and limits established by the administrator and users.  For example,
+and limits established by the administrator and users. For example,
 ephemeral storage does not provide any guarantees of throughput, I/O
 operations/second, or any other measure of storage performance.
 
-== General Storage Guidelines
+== General storage guidelines
 
 A node's local storage can be broken into primary and secondary
-partitions.  Primary partitions are the only ones that can be used for
-ephemeral local storage.  There are two supported primary partitions:
-
-- *Root* -- This partition holds the kubelet’s root directory
-(`/var/lib/kubelet` by default) and `/var/log` directory.  This
-partition may be shared between user pods, OS and {product-title}
-system daemons.  This partition can be consumed by pods via EmptyDir
-volumes, container logs, image layers and container writable layers.
-{product-title} will manage shared access and isolation of this
+partitions. Primary partitions are the only ones that can be used for
+ephemeral local storage. There are two supported primary partitions, root and runtime.
+
+* Root
++
+This partition holds the kubelet’s root directory
+(`/var/lib/kubelet/` by default) and `/var/log/` directory. This
+partition may be shared between user pods, the operating system, and {product-title}
+system daemons. This partition can be consumed by pods by using EmptyDir
+volumes, container logs, image layers, and container writable layers.
+{product-title} manages shared access and isolation of this
 partition.
 
-- *Runtime* -- This is an optional partition which runtimes can use
-for overlay filesystems.  {product-title} will attempt to identify and
+* Runtime
++
+This is an optional partition that runtimes can use
+for overlay file systems. {product-title} attempts to identify and
 provide shared access along with isolation to this partition.
-Container image layers and writable layers are stored here.  If the
-runtime partition exists, the `root` partition will not hold any image
+Container image layers and writable layers are stored here. If the
```

Comment:
maybe s/Container image layers and writable layers are stored here./This partition contains container image layers and writable layers.